src/Microsoft.FeatureManagement/Microsoft.FeatureManagement.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\build\NugetProperties.props" />
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Why target netstandard2.1 here, but netcoreapp3.1 in Microsoft.FeatureManagement.AspNetCore ?

Reply:
asp .net core starting from version 3 supported only in .net core (not in netstandard). Since Microsoft.FeatureManagement does not have asp .net core dependencies it could target netstandard (for Mono or Xamarin I guess), but when we need asp .net core libs it could be only net core.

Reply:
I see. I think it may be better to follow the precedent set by the packages that we are depending on. For example [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging/3.1.5) targets netstandard2.0 and netcoreapp3.1.

Reply:
Sure, no problem. I also think it's better, but decided not to limit availability without a valid reason.